dotnet/src/Extensions/Planning.StepwisePlanner/StepwisePlanner.cs

```diff
@@ -444,9 +438,15 @@ private async Task<string> GetCompletionAsync(IAIService aiService, ChatHistory
         throw new SKException("No AIService available for getting completions.");
     }
 
-    private int GetChatHistoryTokens(ChatHistory chatHistory)
+    private int GetChatHistoryTokens(ChatHistory chatHistory, string? additionalMessage = null, int skipStart = 0, int skipCount = 0)
```

Comment:
nit: We are not getting tokens here, right? We are just counting them. The `CountChatHistoryTokens` method name would convey that better.

Reply:
Or... `GetChatHistoryTokenCount` 😊